src/api/resources/auth_resource.py

```diff
@@ -0,0 +1,37 @@
+from flask import abort
+from flask import request, make_response, current_app
+from flask_restful import Resource, reqparse
+
+from src.common.todo_exception import TodoException
+from src.middleware.auth.auth_store import AuthStore
+
+parser = reqparse.RequestParser()
+parser.add_argument('username')
```

Comment:
maybe it will be better to put username and password in consts so it wont be hard coded several times